packages/salesforcedx-utils-vscode/package.json

```diff
@@ -11,7 +11,7 @@
   ],
   "main": "out/src",
   "dependencies": {
-    "@salesforce/source-tracking": "2.2.24",
+    "@salesforce/source-tracking": "3.1.1",
```

Comment:
Bumped to the latest STL to get https://github.com/forcedotcom/source-tracking/pull/368